src/lib/helpers/date.ts

```diff
@@ -73,6 +73,53 @@ export const toLocaleTimeISO = (datetime: string | number) => {
     return date.toLocaleTimeString('sv');
 };
 
+export const utcHourToLocaleHour = (utcTimeString: string) => {
+    const now = new Date();
+    const [hours, minutes] = utcTimeString.split(':').map(Number);
+    const customTime = Date.UTC(now.getFullYear(), now.getMonth(), now.getDate(), hours, minutes);
+
+    const options: Intl.DateTimeFormatOptions = {
+        hour: 'numeric',
+        minute: 'numeric',
+        hourCycle: 'h23'
+    };
+
+    return new Intl.DateTimeFormat('en', options).format(new Date(customTime));
+};
+
+export const utcWeekDayToLocaleWeekDay = (utcDay: string, utcTimeString: string) => {
+    const now = new Date();
+
+    // easy mapping instead of passing numbers.
+    const dayMap = {
+        Sunday: 0,
+        Monday: 1,
+        Tuesday: 2,
+        Wednesday: 3,
+        Thursday: 4,
+        Friday: 5,
+        Saturday: 6
+    };
```

Comment:
It would be good to have some typing to prevent passing the wrong string in for utcDay.

Reply:
makes sense, a separate type like `WeekDay` for the `utcDay` instead of `string`?